indico/modules/attachments/templates/_attachments.html

```diff
@@ -6,7 +6,7 @@
     {%- endif -%}
 {%- endmacro %}
 
-{% macro _attachment_row(attachment) %}
+{% macro _attachment_row(attachment, short=false) %}
```

Comment:
I was suggesting something like that (iirc my idea was `can_edit`) but he mentioned that it also hides the date. Anyway, I prefer `modification=true`